apps/spruce/src/components/Redirects/WaterfallCommitsRedirect.tsx

```diff
@@ -1,8 +1,46 @@
+import { useEffect } from "react";
 import { useParams, Navigate } from "react-router-dom";
+import { useWaterfallAnalytics } from "analytics";
 import { getWaterfallRoute, slugs } from "constants/routes";
+import { useQueryParam } from "hooks/useQueryParam";
+import { WaterfallFilterOptions } from "pages/waterfall/types";
 
 export const WaterfallCommitsRedirect: React.FC = () => {
   const { [slugs.projectIdentifier]: projectIdentifier } = useParams();
+  const [statusFilters] = useQueryParam<string[]>(
+    WaterfallFilterOptions.Statuses,
+    [],
+  );
+  const [taskFilters] = useQueryParam<string[]>(
+    WaterfallFilterOptions.Task,
+    [],
+  );
+  const [requesterFilters] = useQueryParam<string[]>(
+    WaterfallFilterOptions.Requesters,
+    [],
+  );
+  const [variantFilters] = useQueryParam<string[]>(
+    WaterfallFilterOptions.BuildVariant,
+    [],
```

Comment:
the old commits page uses different query params for example `buildVariants` versus `variants`,  `taskNames` vs `tasks`, `requester` vs `requesters`. does this account for that?

Reply:
Good catch!